### PR TITLE
detect: adds flow.age keyword

### PR DESCRIPTION
**Pull request by catenacyber**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:
https://redmine.openinfosecfoundation.org/issues/5536

Describe changes:
- detect: adds flow.age keyword

I should create a meta ticket to list all the fields logged as integers which should have their detection keyword

Replaces #7841 with review taken into account

### CODE REVIEW

**Comment by suricata-qa** [comment]
WARNING:
 
 | field | baseline | test | % |
|--|--|--|--| 
| | tlpw1_stats_chk | | |
| .tcp.rst | 103033 | 125957 | 122.25% |
| | ips_afp_stats_chk | | |
| .capture.kernel_drops | 0 | 501402 | - |
| .flow.end.state.new | 10800 | 12983 | 120.21% |
| .flow.end.tcp_state.syn_sent | 0 | 916 | - |
| .flow.end.tcp_state.syn_recv | 0 | 208 | - |
| .flow.end.tcp_state.fin_wait1 | 0 | 187 | - |
| .flow.end.tcp_state.fin_wait2 | 0 | 145 | - |
| .flow.end.tcp_state.time_wait | 0 | 133 | - |
| .flow.end.tcp_state.last_ack | 0 | 147 | - |
| .flow.end.tcp_state.close_wait | 0 | 330 | - |
| .flow.end.tcp_liberal | 108000 | 120918 | 111.96% |
| .tcp.reassembly_gap | 108000 | 144487 | 133.78% |
| .tcp.overlap | 0 | 28 | - |
| .app_layer.error.http.parser | 0 | 95 | - |
| .app_layer.error.ftp.gap | 0 | 89 | - |
| .app_layer.error.smtp.gap | 0 | 499 | - |
| .app_layer.error.tls.gap | 0 | 8579 | - |
| .app_layer.error.ssh.gap | 0 | 1 | - |
| .app_layer.error.dcerpc_tcp.parser | 0 | 336 | - |
| .ftp.memuse | 0 | 1942 | - | 

Pipeline 9118

**Comment by victorjulien** [comment]
The QA result is likely due to some randomly appearing variance we're investigating.

File: doc/userguide/rules/flow-keywords.rst [hunk 1]
```diff
@@ -285,3 +285,24 @@ Format
...
+
+ flow.age:3    # exactly 3
+ flow.age:<3   # smaller than 3 seconds
+ flow.age:>=2  # greater or equal than 2 seconds
+
+Signature example::
+
+ alert tcp any any -> any any (msg:"Flow longer than one hour"; flow.age:>3600; flowbits: isnotset, onehourflow; flowbits: onehourflow, name; sid:1; rev:1;)
+
+We use `flowbits` to alert only on the first packet after the flow's age is over one hour.
```
Comment (jufajardini) [inline doc/userguide/rules/flow-keywords.rst:308]: Suggestion:
In this example, we combine `flow.age` and `flowbits` to get an alert on the first packet after the flow's age is older than one hour.

**Review by jufajardini — COMMENTED** [review]
One doc suggestion :)

File: doc/userguide/rules/flow-keywords.rst (hunk 1 above)
Comment (catenacyber) [reply doc/userguide/rules/flow-keywords.rst:308]: Thanks, changing to your proposal

**Comment by catenacyber** [comment]
Replaced by https://github.com/OISF/suricata/pull/7857

**Pull request closed without merging**